pages/edit.js

```diff
@@ -0,0 +1,365 @@
+              <InputAndLabelWrapper>
+                <Select
+                  id="seconds"
+                  value={newTimer.seconds}
+                  onChange={(event) =>
+                    changeTimerChain(
+                      event,
+                      newTimer.id,
+                      newTimerChain,
+                      setNewTimerChain
+                    )
+                  }
+                >
+                  {timeOptions.map((option) => {
+                    return (
+                      <option key={option} value={option}>
+                        {option < 10 ? `0${option}` : option}
+                      </option>
+                    );
+                  })}
+                </Select>
+                <Label htmlFor="seconds"> sec.</Label>
+              </InputAndLabelWrapper>
+            </Li>
+          ))}
+        </FormList>
+        <ButtonWrapper>
+          <AddButton
+            type="button"
+            onClick={() => handleAdd(newTimerChain, setNewTimerChain)}
+          >
+            <Image alt="add" src={add} width="64" height="64" />
+          </AddButton>
+          <DeleteButton
+            type="button"
+            onClick={() => handleRemove(newTimerChain, setNewTimerChain)}
+          >
+            <Image alt="remove" src={remove} width="64" height="64" />
+          </DeleteButton>
+        </ButtonWrapper>
+        <ApplyWrapper>
+          <DummyDiv />
+          <Apply type="submit">
+            <Image alt="hook" src={hook} width="42" height={41.71} />
+            <div>Apply</div>
+          </Apply>
+        </ApplyWrapper>
+      </form>
+      <DownGap />
+    </>
+  );
+}
+
+const HeadlineWrapper = styled.div`
+  position: fixed;
+  width: 100%;
+  backdrop-filter: blur(10px);
+`;
+
+const TopGap = styled.div`
+  height: 75px;
+`;
+
+const FormList = styled.ol`
+  background-color: rgba(52, 108, 61, 0.7);
+  margin: 5px 10vw;
+  border-radius: 5px;
+  padding: 15px 0;
+  list-style-type: none;
+  counter-reset: li;
+`;
+
+const Li = styled.li`
+  display: flex;
+  justify-content: center;
+
+  &::before {
+    counter-increment: li;
+    content: counter(li, decimal-leading-zero);
+    color: #caf6ff;
+    margin: 0 5px 0 0;
+  }
+`;
+
+const InputAndLabelWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+const ExerciseName = styled.input`
+  background-color: #edffdf;
+  margin: 0 1px;
+  border-radius: 5px;
+  border: none;
+  width: 30vw;
+  max-width: 340px;
+`;
+
+const Select = styled.select`
+  text-align: right;
+  background-color: #edffdf;
+  width: 50px;
+  margin: 0 1px;
+  border-radius: 5px;
+  border: none;
+`;
+
+const Label = styled.label`
+  color: #caf6ff;
+  font-weight: 300;
+  font-size: 0.8rem;
+`;
+
+const ButtonWrapper = styled.div`
+  display: flex;
+  justify-content: center;
+`;
+
+const AddButton = styled.button`
+  background-color: transparent;
+  border-radius: 60px;
+  border: none;
```

Comment:
No border but a border radius? Does that do anything?

Reply:
same as above